build.gradle.kts

```diff
@@ -75,6 +79,44 @@ val modlauncherConfig by configurations.register("modlauncher") {
 // create the sourcesets
 val main by sourceSets
 
+val launchJar by tasks.registering(Jar::class) {
+    group = "build"
+    archiveClassifier.set("launch")
+}
+
+val mixinsJar by tasks.registering(Jar::class) {
+    group = "build"
+    archiveClassifier.set("mixins")
+}
+
+val accessorsJar by tasks.registering(Jar::class) {
+    group = "build"
+    archiveClassifier.set("accessors")
+}
+val javadocJar by tasks.registering(Jar::class) {
```

Comment:
Do we really need the JD task for Common? We don't really write them for the impl and so I don't see why we need to potentially generate such a jar.